db/migrate/20230712140342_add_fields_to_posts.rb

```diff
@@ -0,0 +1,8 @@
+class AddFieldsToPosts < ActiveRecord::Migration[7.0]
+  def change
+    # add_column :post, :title, :string
+    # add_column :post, :text, :text
+    # add_column :post, :comments_counter, :integer
+    # add_column :post, :likes_counter, :integer
+  end
```

Comment:
- [ ] I observed you newly added commented-out codes after the last review, kindly ensure that the migration file is not empty for smooth running of the database. 👍